api4/cloud.go

```diff
@@ -67,6 +65,31 @@ func getSubscription(c *Context, w http.ResponseWriter, r *http.Request) {
 	w.Write(json)
 }
 
+func getFreeTierStats(c *Context, w http.ResponseWriter, r *http.Request) {
+	subscription, appErr := c.App.Cloud().GetSubscription()
+
+	if appErr != nil {
+		c.Err = model.NewAppError("Api4.getFreeTierStats", "api.cloud.request_error", nil, appErr.Error(), http.StatusInternalServerError)
+		return
+	}
+
+	count, err := c.App.Srv().Store.User().Count(model.UserCountOptions{})
+	if err != nil {
+		c.Err = model.NewAppError("Api4.getFreeTierStats", "app.user.get_total_users_count.app_error", nil, err.Error(), http.StatusInternalServerError)
+		return
+	}
+	cloudUserLimit := *c.App.Config().ExperimentalSettings.CloudUserLimit
+
+	s := cloudUserLimit - count
+
+	stats, _ := json.Marshal(model.FreeTierStats{
+		RemainingSeats: int(s),
+		TierStatus:     subscription.IsPaidTier,
```

Comment:
Also here for consistency we could keep the same name as `IsPaidTier`
```suggestion
		IsPaidTier:     subscription.IsPaidTier,
```